web-calculator.css

```diff
@@ -0,0 +1,71 @@
+body{
+    background-image: url(https://i.pinimg.com/originals/1e/ef/c5/1eefc5567dc5c55f67ef6849b9235582.jpg), url();
+    background-size: cover;
+    margin:350;
+    position: relative;
```

Comment:
Body is always position relative. You never have to set it because it is the last stop for any absolute positioned elements.